### PR TITLE
Update packages versions

### PR DESCRIPTION
**Pull request by surgupta-msft**
Pipeline run on main branch - https://dev.azure.com/azfunc/internal/_componentGovernance/azure.azure-websites-security?_a=alerts&typeId=26530408&alerts-view-option=active

Pipeline run green on PR branch - https://dev.azure.com/azfunc/internal/_componentGovernance/azure.azure-websites-security?_a=alerts&typeId=29956023&alerts-view-option=active

### CODE REVIEW

File: src/Azure.WebSites.DataProtection/Azure.WebSites.DataProtection.csproj [hunk 1]
```diff
@@ -15,16 +15,18 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.AspNetCore.DataProtection" Version="2.0.0" />
+    <PackageReference Include="Microsoft.AspNetCore.DataProtection" Version="2.1.1" />
     <PackageReference Include="Microsoft.Extensions.DependencyInjection" Version="2.0.0" />
-    <PackageReference Include="System.IdentityModel.Tokens.Jwt" Version="5.1.4" />
+    <PackageReference Include="System.IdentityModel.Tokens.Jwt" Version="5.7.0" />
+    <PackageReference Include="System.Security.Cryptography.Xml" Version="4.7.1" />
   </ItemGroup>
```
Comment (surgupta-msft) [inline src/Azure.WebSites.DataProtection/Azure.WebSites.DataProtection.csproj:22]: ├─ Microsoft.AspNetCore.DataProtection (v2.1.1)
   │  └─ System.Security.Cryptography.Xml (v4.7.1)
   │     └─ System.Security.Permissions (v4.7.0)
   │        └─ System.Windows.Extensions (v4.7.0)
   │           └─ System.Drawing.Common (v4.7.2)

File: src/Azure.WebSites.DataProtection/Azure.WebSites.DataProtection.csproj [hunk 2]
```diff
@@ -15,16 +15,18 @@
...
 
   <ItemGroup Condition=" '$(TargetFramework)' == 'netstandard1.4' ">
     <PackageReference Include="System.Xml.XPath" Version="4.0.1" />
-    <PackageReference Include="System.IdentityModel.Tokens.Jwt" Version="5.1.3" />
+    <PackageReference Include="System.IdentityModel.Tokens.Jwt" Version="5.7.0" />
     <PackageReference Include="System.Text.RegularExpressions" Version="4.3.0" />
     <PackageReference Include="System.Xml.XPath.XDocument" Version="4.0.1" />
+    <PackageReference Include="System.Security.Cryptography.Xml" Version="4.7.1" />
```
Comment (jviau) [inline src/Azure.WebSites.DataProtection/Azure.WebSites.DataProtection.csproj:29]: This shouldn't be needed as it is already included on line 21

File: src/Azure.WebSites.DataProtection/Azure.WebSites.DataProtection.csproj [hunk 3]
```diff
@@ -15,16 +15,18 @@
...
     <PackageReference Include="Microsoft.Extensions.DependencyInjection" Version="2.0.0" />
-    <PackageReference Include="System.IdentityModel.Tokens.Jwt" Version="5.1.4" />
+    <PackageReference Include="System.IdentityModel.Tokens.Jwt" Version="5.7.0" />
+    <PackageReference Include="System.Security.Cryptography.Xml" Version="4.7.1" />
   </ItemGroup>
 
   <ItemGroup Condition=" '$(TargetFramework)' == 'netstandard1.4' ">
     <PackageReference Include="System.Xml.XPath" Version="4.0.1" />
-    <PackageReference Include="System.IdentityModel.Tokens.Jwt" Version="5.1.3" />
+    <PackageReference Include="System.IdentityModel.Tokens.Jwt" Version="5.7.0" />
```
Comment (jviau) [inline src/Azure.WebSites.DataProtection/Azure.WebSites.DataProtection.csproj:26]: This is already included on line 20

File: src/Azure.WebSites.DataProtection/Azure.WebSites.DataProtection.csproj [hunk 4]
```diff
@@ -36,7 +38,7 @@
 
   <ItemGroup Condition="'$(TargetFramework)' == 'net46'">
```
Comment (jviau) [inline src/Azure.WebSites.DataProtection/Azure.WebSites.DataProtection.csproj:39]: This `ItemGroup` can be deleted, it is already included on line 20

File: src/Azure.WebSites.DataProtection/Azure.WebSites.DataProtection.csproj (hunk 3 above)
Comment (surgupta-msft) [reply src/Azure.WebSites.DataProtection/Azure.WebSites.DataProtection.csproj:26]: Isn't line 20 for netstandard2.0 and this one for netstandard1.4? Before my update also, they had this package at 2 places.

File: src/Azure.WebSites.DataProtection/Azure.WebSites.DataProtection.csproj (hunk 2 above)
Comment (surgupta-msft) [reply src/Azure.WebSites.DataProtection/Azure.WebSites.DataProtection.csproj:29]: same for this

File: src/Azure.WebSites.DataProtection/Azure.WebSites.DataProtection.csproj (hunk 4 above)
Comment (surgupta-msft) [reply src/Azure.WebSites.DataProtection/Azure.WebSites.DataProtection.csproj:39]: This is for net46.

File: test/Azure.WebSites.DataProtection.UnitTests/Azure.WebSites.DataProtection.UnitTests.csproj [hunk 5]
```diff
@@ -5,12 +5,14 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.AspNetCore.DataProtection" Version="2.0.0" />
+    <PackageReference Include="Microsoft.AspNetCore.DataProtection" Version="2.1.1" />
+    <PackageReference Include="System.Security.Cryptography.Xml" Version="4.7.1" />
+    <PackageReference Include="System.Drawing.Common" Version="4.7.2" />
     <PackageReference Include="Microsoft.Extensions.DependencyInjection" Version="2.0.0" />
-    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="15.3.0" />
+    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="15.9.2" />
```
[the remark this answers is not in the document]
Comment (surgupta-msft) [reply test/Azure.WebSites.DataProtection.UnitTests/Azure.WebSites.DataProtection.UnitTests.csproj]: sounds good

File: src/Azure.WebSites.DataProtection/Azure.WebSites.DataProtection.csproj (hunk 3 above)
Comment (jviau) [reply src/Azure.WebSites.DataProtection/Azure.WebSites.DataProtection.csproj:26]: No, line `20` and `21` has no condition on the `ItemGroup`, so it applies to all TFMs.

I imagine this was a mistake before your change to have the package in both places

File: src/Azure.WebSites.DataProtection/Azure.WebSites.DataProtection.csproj (hunk 2 above)
Comment (jviau) [reply src/Azure.WebSites.DataProtection/Azure.WebSites.DataProtection.csproj:29]: See my other comment. The `ItemGroup` at line 17 applies to all TFMs. In fact, this project only supports `netstandard2.0` - there isn't even a netstandard1.4 or net46 TFM anymore

Comment (surgupta-msft) [reply src/Azure.WebSites.DataProtection/Azure.WebSites.DataProtection.csproj]: Ya this makes more sense and will cleanup this file. Thanks for the suggestion.

**Review by jviau — APPROVED** [review]

**Pull request merged**